src/utils/device/api.ts

```diff
@@ -252,21 +252,26 @@ export class DeviceAPI {
 			return await new Promise((resolve, reject) => {
 				return request(opts, (err, response, body) => {
 					if (err) {
-						return reject(err);
+						reject(err as Error);
+						return;
 					}
 					switch (response.statusCode) {
-						case 200:
-							return resolve(body);
-						case 400:
-							return reject(
-								new ApiErrors.BadRequestDeviceAPIError(body.message),
-							);
-						case 503:
-							return reject(
-								new ApiErrors.ServiceUnavailableAPIError(body.message),
-							);
-						default:
-							return reject(new ApiErrors.DeviceAPIError(body.message));
+						case 200: {
+							resolve(body);
+							return;
```

Comment:
nit: I think more common practice is to use `break` in a switch, but maybe I'm mistaken

Reply:
Would rather avoid touching potentially breaking runtime code on a linter bum